common/src/main/java/org/apache/gravitino/dto/authorization/RoleDTO.java

```diff
@@ -43,9 +44,13 @@ public class RoleDTO implements Role {
   @JsonProperty("properties")
   private Map<String, String> properties;
 
+  @Nullable
   @JsonProperty("securableObjects")
   private SecurableObjectDTO[] securableObjects;
 
+  @JsonProperty("securableObjectsCount")
+  private int securableObjectsCount;
```

Comment:
Maybe we can also change to `count` here? Also, what is the serialization result of this field, "0" or "null'?